samples/hcls_2015.shex

```diff
@@ -97,7 +97,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     dct:title rdf:langString,
     ###dct:alternative rdf:langString, is a MAY property
     dct:description rdf:langString,
-    (dct:creator IRI | pav:createdBy IRI)+,
+    (dct:creator IRI | pav:createdBy IRI)?,
```

Comment:
This is clearly wrong, but I don't understand why its not passing. Could well be a problem in the data rather than the schema.